README.md

```diff
@@ -1,739 +1,83 @@
-                        The directory to store the extracted data. This is
-			optional and defaults to the execute directory.
-			This flag is only used if the extract-data flag is set.
-  -m MATLAB_DIR, --matlab-dir MATLAB_DIR
-                        The directory to store the modified case.mat used by
-			the engine. This is optional and defaults to the execute
-			directory. This flag is only used if the extract-data
-			flag is set.
-  -k, --keep-matlab     The result.mat files found in the execute directory will
-      			be kept instead of deleted after extraction. This flag
-			is only used if the extract-data flag is set.
-```
 
-### Compatibility with PowerSimData
+# REISE.jl
+**REISE.jl** is the Renewable Energy Simulation Engine developed by [Breakthrough
+Energy Sciences](https://science.breakthroughenergy.org/) (BES) to solve DCOPF
+problems. It is written and Julia and can be used with the BES software ecosystem (see
+[PowerSimData]) or in a standalone
+mode (see [Zenodo](https://doi.org/10.5281/zenodo.4538590) for sample data).
 
-Within the Python code in this repo, there is some code to maintain
-compatibility with the `PowerSimData` framework.
 
-Both `call.py` and `extract_data.py` can be called using a positional
-argument that corresponds to a scenario id as generated by the
-`PowerSimData` framework. Using this invocation assumes the presence
-of the `PowerSimData` infrastructure including both a Scenario List
-Manager and Execute List Manager. This option is not intended for manual
-simulation runs.
+## Main Features
+Here are a few things that **REISE.jl** can do:
+* Formulate Production Cost Model (PCM) into an optimization problem that can be solved
+  by professional solvers compatible with [JuMP]
+* Decompose a model that couldn't be solved all at once in memory (due to high spatial
+  or temporal resolution or both) into a series of shorter-timeframe intervals that are
+  automatically run sequentially.
+* Handle storage and demand flexibility
+* Handle adaptive infeasible/suboptimal/numeric issues via the homogenous barrier
+  algorithm (when using the [Gurobi] solver) and involuntary load shedding
 
-Note also the different naming convention for various directories by
-`PowerSimData` as compared to the options for the Python scripts within
-this repository.
 
-## Docker
+## Where to get it
+For now, only the source code is available. Clone or Fork the code here on GitHub.
 
-The easiest way to setup this engine is within a Docker image.
 
-There is an included `Dockerfile` that can be used to build the Docker image. With the Docker daemon installed and running, navigate to the `REISE.jl` folder containing the `Dockerfile` and build the image:
+## Dependencies
+**REISE.jl** relies on several Julia packages. The list can be found in the
+***Project.toml*** file located at the root of this package.
 
-```bash
-docker build . -t reisejl
-```
+Note that it is possible to use [Gurobi] as a solver.
```

Comment:
This would probably be a good place to mention that specific solver dependencies need to be installed separately. Also, since we also seem to explicitly support `GLPK` and `Clp`, does it make sense to list them here in addition to `Gurobi`?

Reply:
I spent quite some time on this. My understanding is that you don't need to install Gurobi with the Docker installation. You will need a license but that's it. Without going too much into the details (we do that in the docs), how can we say a few words about solvers? What about: 

Several solvers, commercial (Gurobi) and open source (GLPK and Clp) can be used to solve the DCOPF problem. See our website for more details

Reply:
How about something like: "This program builds an optimization problem, but still relies on an external solver to generate results. Any solver compatible with JuMP can be used, although performance with open-source solvers (e.g. Clp, GLPK) may be significantly slower than with commercial solvers". I don't think we need to mention any commercial solver in particular.

Reply:
Done